Unofficial/UpDog-Credit-Union/account/index.php

```diff
@@ -0,0 +1,271 @@
+<?php session_start(); ?>
+
+<!doctype html>
+<html lang="en"> 
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+
+	<link rel="stylesheet" href="/css /beststyle2018.css"> 
+    <title>Your UpDog Account Details</title>
+	
+	<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
+	<link rel="icon" href="/favicon.ico" type="image/x-icon">
+	
+	
+  </head>
+  <body>
+	
+	<?php 
+	$_SESSION['extra'] =  '<li class="nav-item"><a class="nav-link" href="#">Log Out</a></li>';

```

Comment:
Somewhere, whether it be in `header.php` or just here or around here, it actually prints out the PHP of the page.

As shown [here](https://screenshots.firefox.com/ZcaeGbcv2BjQAvLt/localhost)

idk if it's me, or something that's in the code. If it's just me, ignore